continuationsPlugin/src/test/scala/continuations/ContinuationsPluginSuite.scala

```diff
@@ -308,7 +308,7 @@ class ContinuationsPluginSuite extends FunSuite, CompilerFixtures, StateMachineF
         case (tree, _) =>
```

Comment:
should we maybe add a test here of what was not working before that now works?

Reply:
It's covered by the ListMapNoResume example.

Reply:
yes but we are not running all the examples one by one every time we open a PR, are we?